### PR TITLE
add setup script to be used for setting up a UAT docker leeloo

### PR DESCRIPTION
**Pull request by feedmypixel**
This script is used to setup a leeloo docker which is used within the `user_acceptance_tests` job in Data Hub frontends CircleCi workflow.

More information on how this is being used can be found in [#752](https://github.com/uktrade/data-hub-frontend/pull/752)

### CODE REVIEW

File: setup-uat.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
+# This file is used as a cmd script with an automatically built backend docker to run User Acceptance Tests against on circleCI.
+# For more information about how this is used please see https://github.com/uktrade/data-hub-frontend
+
+dockerize -wait ${POSTGRES_URL} -wait ${ES5_URL} -timeout 60s
+python /app/manage.py migrate
+python /app/manage.py loadmetadata
+python /app/manage.py load_omis_metadata
+python /app/manage.py createinitialrevisions
```
Comment (reupen) [inline setup-uat.sh]: I'd have thought createinitialrevisions should be after loaddata. Though the README has them this way round right now...

File: setup-uat.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+echo "from oauth2_provider.models import Application; from datahub.oauth.models import OAuthApplicationScope; app = Application.objects.create(name='circleci', client_id='${API_CLIENT_ID}', client_secret='${API_CLIENT_SECRET}', client_type=Application.CLIENT_CONFIDENTIAL, authorization_grant_type=Application.GRANT_PASSWORD); OAuthApplicationScope.objects.create(application=app, scopes=['internal-front-end'])" | /app/manage.py shell
```
Comment (reupen) [inline setup-uat.sh]: Is there a way to make these lines more readable (spread them over multiple lines)? Then maybe later on they could be converted to manage.py commands (with parameters).

File: setup-uat.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+echo "from oauth2_provider.models import Application;
+      from datahub.oauth.models import OAuthApplicationScope;
+      app = Application.objects.create(
+        name='circleci',
+        client_id='${API_CLIENT_ID}',
+        client_secret='${API_CLIENT_SECRET}',
+        client_type=Application.CLIENT_CONFIDENTIAL,
+        authorization_grant_type=Application.GRANT_PASSWORD
+      );
+      OAuthApplicationScope.objects.create(application=app, scopes=['internal-front-end'])" | /app/manage.py shell
```
Comment (marcofucci) [inline setup-uat.sh]: Wouldn't this raise `IndentationError`? Have you tried running it?

File: setup-uat.sh (hunk 1 above)
Comment (reupen) [reply setup-uat.sh]: I meant after loaddata (rather than loadmetadata)

File: setup-uat.sh (hunk 3 above)
Comment (feedmypixel) [reply setup-uat.sh]: Yes it did 😄  (pyton noob error) just confirmed that this is running and passing in [circleCi])https://circleci.com/gh/uktrade/data-hub-frontend/7434)

**Comment by feedmypixel** [comment]
All passing:

screen shot 2017-10-10 at 17 03 37

**Pull request merged**